src/main/java/chess/controller/ChessGameController.java

```diff
@@ -0,0 +1,98 @@
+package chess.controller;
+
+import chess.domain.BoardFactory;
+import chess.domain.ChessGame;
+import chess.domain.piece.Position;
+import chess.view.InputView;
+import chess.view.OutputView;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ChessGameController {
+    private static final Pattern MOVE_COMMAND_PATTERN = Pattern.compile("^move\\s+([a-h][1-8]\\s+[a-h][1-8])$");
+
+    private final InputView inputView;
+    private final OutputView outputView;
+
+    public ChessGameController(InputView inputView, OutputView outputView) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+    }
+
+    public void run() {
+        outputView.printWelcomeMessage();
+        waitStartCommand();
+    }
+
+    private void waitStartCommand() {
+        String command = inputView.readCommand().trim();
+        if (command.equals("start")) {
```

Comment:
```suggestion
        if ("start".equals(command)) {
```
이렇게 해보면 어떤 차이가 있을까요?

Reply:
command에 null 이 들어와도 NPE 가 발생하지 않을것 같아요.  반영하겠습니다!